modules/core/04-channel/v2/types/msgs.go

```diff
@@ -13,6 +15,8 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v9/modules/core/errors"
 )
 
+const MaxTimeoutDelta time.Duration = 24 * time.Hour
```

Comment:
why was 24 hours chosen?

Reply:
This decision came after discussion with Adi. While we have a median time for MsgReceivePacket of 17s and for Ack 11s, we decided to set the timeout to 24h to take in consideration potential relayers/Sp1 network downtimes. Especially relayers in some cases have gone down for periods of time of 2-4 hours. 

In case you have a different opinion/guidance on why this should be different we can revisit this choice.

Reply:
there's https://github.com/cosmos/ibc-go/issues/7530 that is pertinent too

Reply:
I think 24 hours is excessive, its the time your tokens can be left in limbo and you can't really do anything about it, I would say something more like 6 hours is even being generous

Reply:
note that this is the max value a user could allow for a packet. Think we can easily change to six though, I have no insight into what value is typically used for timeouts atm

Reply:
Yeah, I initially proposed a 4h too, but we kept it in 24h to be conservative. Anyway, as Jim says, that's easy to change. So let's agree on a concrete value.